mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -5,18 +5,24 @@ CLASS net/minecraft/class_127 net/minecraft/entity/LivingEntity
 	FIELD field_1033 movementSpeed F
 	FIELD field_1036 health I
 	FIELD field_1038 hurtTime I
+	FIELD field_1041 deathTime I
 	FIELD field_1042 attackCooldown I
+	FIELD field_1059 despawnCounter I
+	METHOD method_909 getHeldItem ()Lnet/minecraft/class_31;
 	METHOD method_910 tickLiving ()V
 	METHOD method_911 getRandomSound ()Ljava/lang/String;
 	METHOD method_912 getHurtSound ()Ljava/lang/String;
 	METHOD method_913 getDeathSound ()Ljava/lang/String;
 	METHOD method_914 getDroppedId ()I
 	METHOD method_915 getSoundVolume ()F
+	METHOD method_916 getLimitPerChunk ()I
```

Comment:
Is this really the limit per chunk? It looks like its the number in each spawn group. It looks like modern yarn does have this function. But, I am uncertain if it is the same thing or not
`
                                                        livingEntity31.setPositionAndAnglesKeepPrevAngles(float28, float29, float30, world.field_214.nextFloat() * 360.0f, 0.0f);
                                                        if (livingEntity31.canSpawn()) {
                                                            ++integer21;
                                                            world.spawnEntity(livingEntity31);
                                                            method_1872(livingEntity31, world, float28, float29, float30);
                                                            if (integer21 >= livingEntity31.getLimitPerChunk()) {
                                                                continue Label_0253;
                                                            }
                                                        }
                                                        integer4 += integer21;
`